docs/agented-vm-guide.md

```diff
@@ -212,21 +212,27 @@ documentation.
 
 ### Installation On Linux VMs
 
-Download the [wrapper install script](../hack/install-wrapper.sh) and run
-installation.
+Download the [wrapper install script](../hack/install-vm-agent-wrapper.sh) from a
+[list of nephe releases](https://github.com/antrea-io/nephe/releases) and run
+the installation. For any given release <TAG> (e.g. v0.2.0), you can download
+the script as follows:
 
 ```bash
-./install-wrapper.sh --ns vm-ns --antrea-version v1.9.0 --kubeconfig ./antrea-agent.kubeconfig \
-  --antrea-kubeconfig ./antrea-agent.antrea.kubeconfig --bin ./antrea-agent
+curl https://github.com/antrea-io/nephe/releases/download/<TAG>/install-vm-agent-wrapper.sh --output install-vm-agent-wrapper.sh
```

Comment:
@Anandkumar26  break this into 2. For downloading release version, do this, and for TOT do something else, like we do for nephe.yml

Reply:
okay.